### PR TITLE
Selectively disable cache for metadata enrichment calls

### PR DESCRIPTION
**Pull request by gizmo-rt**
##### Description

https://github.com/SumoLogic/sumologic-kubernetes-fluentd/pull/403

---

##### Checklist

Remove items which don't apply to your PR.

- [x] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by swiatekm-sumo — CHANGES_REQUESTED** [review]
Changes look good to me. Can you reformat your commits according to https://www.conventionalcommits.org/en/v1.0.0/? It's what we follow in this project.

File: CHANGELOG.md [hunk 1]
```diff
@@ -175,6 +175,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - chore: bump Thanos image to our build of v0.23.1 [#1973][#1973]
 - Introduced option to add cache refresh delay for metadata enrichment calls [#1974][#1974]
 - chore(deps): bump Sumo OT distro to 0.0.47-beta.0 [#2035][#2035]
+- Introduced option to add selectively disable cache for metadata enrichment calls [#2190][#2190]
```
Comment (pmalek-sumo) [inline CHANGELOG.md:178]: You'll need a link added below like for other PRs.

**Comment by gizmo-rt** [comment]
> Changes look good to me. Can you reformat your commits according to https://www.conventionalcommits.org/en/v1.0.0/? It's what we follow in this project.

@swiatekm-sumo Are we not doing squash and merge ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - feat: add pprof extension for otelcol [#2173][#2173]
+- Introduced option to add selectively disable cache for metadata enrichment calls [#2190][#2190]
```
Comment (perk-sumo) [inline CHANGELOG.md:13]:
```suggestion
- feat: option to add selectively disable cache for metadata enrichment calls [#2190][#2190]
```

File: deploy/helm/sumologic/values.yaml [hunk 3]
```diff
@@ -261,7 +261,7 @@ sumologic:
 fluentd:
   image:
     repository: public.ecr.aws/sumologic/kubernetes-fluentd
-    tag: 1.14.4-sumo-1
+    tag: 1.14.5-sumo-1
```
Comment (perk-sumo) [inline deploy/helm/sumologic/values.yaml:264]: This needs to be in a separate PR / commit with changes to support matrix as well.

**Review by perk-sumo — CHANGES_REQUESTED** [review]
Needs to change the support matrix as well after the Fluentd bump.

**Comment by perk-sumo** [comment]
Please rebase your PR on top of this one: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2196

**Review by perk-sumo — APPROVED** [review]
Ok, I've rebased and it's ready to be merged with squash now.
@swiatekm-sumo you agree?

**Review by swiatekm-sumo — APPROVED** [review]
